driver/main.ml

```diff
@@ -41,7 +41,26 @@ module Options = Main_args.Make_bytecomp_options (struct
   let _dllpath s = dllpaths := !dllpaths @ [s]
   let _for_pack s = for_package := Some s
   let _g = set debug
-  let _i () = print_types := true; compile_only := true
+  let _i () =
+    print_types := true;
+    compile_only := true;
+    stop_after := Some Compiler_pass.Typing;
+    ()
+  let _stop_after pass =
+    let module P = Compiler_pass in
+    begin match P.of_string pass with
+    | None ->
+        let error msg = raise (Arg.Bad msg) in
```

Comment:
Note that this code can never be reached now that we use `Symbol`, so we could effectively just use `assert false`